### PR TITLE
Sanitize input string in `omerotools.parse_url`

### LINKED ISSUE #115: Method crashes if link ends with a space
https://github.com/imcf/python-imcflibs/blob/26f64c95a9ee187d08f0ff3cdb98591d4df432cb/src/imcflibs/imagej/omerotools.py#L30

### PR DESCRIPTION
**Pull request by rohangirishrao**
Fixes from #115: the issue of a trailing white space leading to the method crashing, and gracefully raises an error if the input string is empty.

### CODE REVIEW

File: src/imcflibs/imagej/omerotools.py [hunk 1]
```diff
@@ -57,6 +57,12 @@ def parse_url(client, omero_str):
     >>> for wrapper in img_wrappers:
     >>>    imp = wpr.toImagePlus(client)
     """
+    if omero_str is None or str(omero_str).strip() == "":
```
Comment (ehrenfeu) [inline src/imcflibs/imagej/omerotools.py]: I think the conditions can be simplified to this:

```Python
if not str(omero_str).strip():
```

➡️ we don't need to test explicitly for `None` (or is there a good reason?), and the *pythonic* way to check if a string is empty is by simply using it as a boolean.

**Review by ehrenfeu — CHANGES_REQUESTED** [review]

File: src/imcflibs/imagej/omerotools.py (hunk 1 above)
Comment (rohangirishrao) [reply src/imcflibs/imagej/omerotools.py]: Hmm, fair. Probably more robust to check for `None`, but seeing as the script parameter will always be a string input, we can just check for an empty string.

Comment (ehrenfeu) [reply src/imcflibs/imagej/omerotools.py]: In the end you're only gaining something if it made a (functional) difference by checking for `None`.

If something calls the function with `None` instead of a `str` type, they will anyway get a `TypeError` from the `str()` call in the conditional.

So unless there is an explicit reason to treat the `None` case differently (which the current conditional anyway doesn't do), it's preferable to have less complex code (readability, testability, ...).

**Review by ehrenfeu — APPROVED** [review]

**Comment by ehrenfeu** [comment]
Add to changelog #126
